gravitee/values.yaml

```diff
@@ -151,17 +165,21 @@ ui:
     enabled: true
     # Used to create an Ingress record.
     hosts:
-      - developer.example.com
+      - apim.example.com
     annotations:
       kubernetes.io/ingress.class: nginx
       kubernetes.io/app-root: /management
       kubernetes.io/rewrite-target: /management
       ingress.kubernetes.io/configuration-snippet: "etag on;\nproxy_pass_header ETag;\n"
-    tls:
+    #tls:
```

Comment:
?

Reply:
I kept the comment to help people understand that the cert must be created in the namespace.
We can move it to the README file though. What're your cents?

Reply:
Agree. I will change.